test/com/google/javascript/jscomp/NewTypeInferenceES5OrLowerTest.java

```diff
@@ -8787,14 +8787,13 @@ public void testMeetOfLooseObjAndNamedDoesntCrash() {
         "};"));
   }
 
-  // public void testVarargs() {
-  //   // TODO(blickly): Investigate why this doesn't pass
-  //   typeCheck(LINE_JOINER.join(
-  //       "function foo(baz, /** ...number */ es6_rest_args) {",
-  //       "  var bar = [].slice.call(arguments, 0);",
-  //       "}",
-  //       "foo(); foo(3); foo(3, 4);"));
-  // }
+   public void testVarargs() {
+     typeCheck(LINE_JOINER.join(
+         "function foo(baz, /** ...number */ es6_rest_args) {",
+         "  var bar = [].slice.call(arguments, 0);",
```

Comment:
IIRC this is not what the transpiler outputs anymore but it's still a valid test.